test/python/pulse/test_transforms.py

```diff
@@ -319,7 +323,7 @@ def test_padding_until_greater(self):
             sched
             | Delay(delay, DriveChannel(0))
             | Delay(30, DriveChannel(0)).shift(20)
-            | Delay(40, DriveChannel(1)).shift(10)
+            | Delay(40, DriveChannel(1)).shift(10)  # pylint: disable=unsupported-binary-operation
         )
```

Comment:
This suppression looks weird - there's no binary operation on this line.

Reply:
Agreed. As with the last one, it only happened in CI, and not locally.